### DISCUSSION TITLE
How can I obtain the coordinates for the bounding box of a model (min and max positions of a model)?

### DISCUSSION THREAD

**Discussion by mjmashayekhi** [ansys/pyaedt#4305] [Q&A] [answered]
I have imported a tzr model to Icepak AEDT. I need a script to obtain it's bounding box coordinates to be used to create a new box based on that.

**Comment by Samuelopez-ansys** [comment, marked as answer]
@mjmashayekhi Just doing:

app = pyaedt.Icepak()

design_bounding_box = app.modeler.obounding_box

If you have multiple objects, you can navigate object by object and get this information:
<img width="608" alt="image" src="https://github.com/ansys/pyaedt/assets/85613111/123731bd-2643-4e92-9875-15dc97ebd45b">
